build-scripts/hack/sync-registry.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+DIR=$(realpath $(dirname "${0}"))
+
+docker run -v "${DIR}/registry-k8s-io.yaml":/config.yaml quay.io/skopeo/stable:v1.15 sync \
+  --src yaml \
+  --dest docker \
+  /config.yaml ghcr.io/canonical \
+  --format oci \
+  --dest-creds "${ACTOR}":"${TOKEN}"
```

Comment:
Let's make this a `$REGISTRY_CREDS` instead, so that it matches the scopeo arg

```suggestion
  --dest-creds "${DEST_CREDS}"
```